packages/satex/satex.0.1.1/opam

```diff
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "String diagram generator for LaTeX"
+description:
+  "Satex takes descriptions of string diagrams in the form of categorical compositions and converts them to string diagrams drawn in TikZ."
+maintainer: ["Samuel Mimram <smimram@gmail.com>"]
+authors: ["Samuel Mimram <smimram@gmail.com>"]
+license: "GPL-3.0"
+homepage: "https://github.com/smimram/satex"
+bug-reports: "https://github.com/smimram/satex/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "menhir" {build}
```

Comment:
```suggestion
  "menhir" {>="20180528"}
```

for infer-write-query use.  You also need to be careful of {build} here -- if Menhir is recompiled and generates a new automata, then you will need to rebuild this package. I'll remove the build tag from the published opam release.